interfaces/acados_template/acados_template/acados_ocp_options.py

```diff
@@ -622,11 +740,23 @@ def nlp_solver_tol_comp(self):
     def nlp_solver_max_iter(self):
         """
         NLP solver maximum number of iterations.
-        Type: int > 0
+        Type: int >= 0
         Default: 100
         """
         return self.__nlp_solver_max_iter
 
+    @property
+    def nlp_solver_terminate_after_small_step(self):
```

Comment:
Can we just use the other option `nlp_solver_tol_min_step_norm` and deduct from it if we want to terminate?
i.e. `nlp_solver_tol_min_step_norm > 0` means we want to terminate after small step norm, and otherwise, we don't check the step norm as is now.